### ISSUE TITLE
1.5.2 setup.py requires 'requests'

### ISSUE THREAD

**Issue by kurtiss** [logdna/python#43]
It seems there was a change to the imports in setup.py that eventually requires the 'requests' library to be installed at setup time. Here's my stacktrace:

```
Downloading https://files.pythonhosted.org/packages/c9/80/fd88bb8e30285ff3f1c3917ac7d6d4b9d2ff2a4017063959c838f0f1ecd0/logdna-1.5.2.tar.gz
108     ERROR: Command errored out with exit status 1:
109      command: /builds/…/3.7.2/venv/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-wheel-7zde05dd/logdna/setup.py'"'"'; __file__='"'"'/tmp/pip-wheel-7zde05dd/logdna/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base pip-egg-info
110          cwd: /tmp/pip-wheel-7zde05dd/logdna/
111     Complete output (9 lines):
112     Traceback (most recent call last):
113       File "<string>", line 1, in <module>
114       File "/tmp/pip-wheel-7zde05dd/logdna/setup.py", line 6, in <module>
115         import logdna._version as _version
116       File "/tmp/pip-wheel-7zde05dd/logdna/logdna/__init__.py", line 1, in <module>
117         from .logdna import LogDNAHandler
118       File "/tmp/pip-wheel-7zde05dd/logdna/logdna/logdna.py", line 3, in <module>
119         import requests
120     ModuleNotFoundError: No module named 'requests'
```

**Comment by abmohan** [comment]
I'm experiencing this as well when deploying to Heroku. 

Strangely, when I run `heroku bash` and try pip installing, it works. It only seems to fail during the build process, even when I explicitly mention `requests` in my requirements.txt file (and above `logdna`, too).

**Comment by smusali** [comment]
Hi @kurtiss and @abmohan,

Thanks for reporting the bug!

We are working on fixing this issue on #45 - we will release the fix and let you know asap!

Have a good day,
Samir

**Comment by vilyapilya** [comment]
@kurtiss  @abmohan  the fix is in the next version 1.5.3.
